tests/test_transpilers_placer.py

```diff
@@ -158,19 +158,19 @@ def test_random(reps):
 
 
 @pytest.mark.parametrize("gates", [None, 5, 13])
-def test_backpropagation(gates):
+def test_reverse_traversal_technique(gates):
     circuit = star_circuit()
     connectivity = star_connectivity()
     routing = ShortestPaths(connectivity=connectivity)
-    placer = Backpropagation(connectivity, routing, depth=gates)
+    placer = ReverseTraversalTechnique(connectivity, routing, depth=gates)
     layout = placer(circuit)
     assert_placement(circuit, layout)
 
 
-def test_backpropagation_no_gates():
+def test_ReverseTraversalTechnique_no_gates():
```

Comment:
Well, I thought it was just the same, but maybe you haven't noticed from the interface: even this has an inconsistent case.